src/managers/GuildManager.js

```diff
@@ -50,7 +50,7 @@ class GuildManager extends BaseManager {
   /**
    * Partial data for a Role.
    * @typedef {Object} PartialRoleData
-   * @property {number} [id] The ID for this role, used to set channel overrides,
+   * @property {Snowflake} [id] The ID for this role, used to set channel overrides,
```

Comment:
I said this can also be a number for a reason! You can specify a role as `{ id: 1, color: 0xffffff }` or `{ id: "8000000000000000000", role: 0x000000 }`. The snowflake type is ``type Snowflake = `${bigint}`;``, which is a string that can be a bigint

Ditto for the rest

Reply:
You said, "it can be a `Snowflake` too", so I thought `number` is wrong type and changed it to `Snowflake` only, which is what we have as a type for most IDs. Fixed it in the latest commit.